content/blog/2023/web-pwa-native-RWA1.md

```diff
@@ -0,0 +1,136 @@
+---
+title: Part 1 - Responsive Web App vs PWA vs Native Mobile App
+sub: When to choose a responsive web app (RWA) for your digital product
+author: sondra
+date: 2023-05-05
+tags:
+  - Article
+  - Business Development
+  - Push Notifications
+  - Recommendations
+  - Startups
+  - User Experience
+image:
+  src: blog/2023/part1-web.jpg
+  alt: |
+    spider web with dew drops
+summary: |
+  The decision of what platform
+  to build your app with
+  is quite important –
+  affecting project scope, timeline, and budget.
+  But understanding the differences between
+  a responsive web app,
+  a progressive web app,
+  or a native mobile app,
+  and which one is right for your project
+  can seem insurmountably difficult.
+  Welcome to part 1 of a three-part series
+  unpacking the reasons to choose
+  one platform over another.
+  Let's start with the responsive web app (RWA).
+---
+{% import 'embed.macros.njk' as embed %}
+
+You have an idea for a custom digital product you’d like to build, and it’s time to decide how to build it. One of your friends swears you won’t succeed unless you build a native mobile app for iPhone. A colleague argues that a responsive web app is the only way to go. Who’s right? Are there alternative options? The decision of how to build your app is quite important – affecting project scope, timeline, and budget – but understanding when to choose one option over another can seem insurmountably difficult.
```

Comment:
@SondraE The phrase "insurmountably difficult" was mentioned at the end of the previous paragraph, too. I don't see the word insurmountably often so it really stood out to me to use it so soon after the first. I don't think we lose anything by dropping it in one of the places.